test/integration-test.sh

```diff
@@ -6,33 +6,54 @@ go build ./cmd/stupid
 curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh | bash
 
 cd ./fabric-samples/
-if [ $1 == "1.4" ]; then
-  git checkout release-1.4
-  cd first-network
-  echo y | ./byfn.sh up -i 1.4.8
-  # comments here for 1.4.8 work around as docker image issue.
-  # docker pull hyperledger/fabric-orderer:amd64-1.4  
-  # Error response from daemon: manifest for hyperledger/fabric-orderer:amd64-1.4 not found: manifest unknown: manifest unknown
-  cp -r crypto-config "$DIR"
-  
-  CONFIG_FILE=./test/config14org1andorg2.yaml
-fi
+case $1 in
+ mock)
+    #This is a tiny, mocked Fabric that does **nothing**, so it runs really fast. 
+    #We use this tool to test the limit of `stupid`, and making sure there's no performance regression. 
+    #By Aug 2020, `stupid` could produce ~15k tps.
+    cd test-network
+    echo y |  ./network.sh down 
+    echo y |  ./network.sh up
+    cp -r organizations "$DIR"
+    cd "$DIR"
+    CONFIG_FILE=./test/configmock.yaml
+    go build ./mock/mockfabric
+    nohup ./mockfabric 2>&1 &
+    ;;
+ 14)
+    git checkout release-1.4
```

Comment:
do we have to checkout `release-1.4` branch? doesn't `-i 1.4` work?

i did not check and it's very likely that you are correct here. Just want to confirm

Reply:
as test network remove 1.4 support.
ref:

https://github.com/hyperledger/fabric-samples/blob/master/test-network/network.sh#L95